toolkit/tools/pkg/imagecustomizerlib/customizeverity_test.go

```diff
@@ -120,8 +120,8 @@ func testCustomizeImageVerityShrinkExtractHelper(t *testing.T, testName string,
 	config.Storage.FileSystems[hashPartitionNum-1].Type = "ext4"
 
 	// Customize image, shrink partitions, and split the partitions into individual files.
-	err = CustomizeImage(buildDir, testDir, &config, baseImage, nil, outImageFilePath, "", "raw",
-		"" /*outputPXEArtifactsDir*/, true /*useBaseImageRpmRepos*/, true /*enableShrinkFilesystems*/)
+	err = CustomizeImage(buildDir, testDir, &config, baseImage, nil, outImageFilePath, "cosi",
+		"" /*outputPXEArtifactsDir*/, true /*useBaseImageRpmRepos*/)
 	if !assert.NoError(t, err) {
 		return
 	}
```

Comment:
The `image_XX.raw` file no longer exist when the output is COSI. You need to extract them from the COSI file.